### DISCUSSION TITLE
How to change Advisers category?

### DISCUSSION THREAD

**Discussion by jmcca18** [joaomlourenco/novathesis#328] [Q&A]
Hello,

I would like to add an advisor sub category to my master thesis. In the template we can choose between advisers and co-advisers, but my professor requested to add a 3rd member, and he needs to be labeled as a monitor. Is it possible to add a 3rd sub category?

Thank you!

**Comment by joaomlourenco** [comment]
@jmcca18 
That can be done, but is non-standard.
Please let me know your school and the degree, and I'll check how to make it (if possible) painlessly! :)

**Reply by jmcca18** [reply]
Hello, sorry for the late reply!

My school is NOVA FCT and it is a Master/BSc degree. 

Thanks for the help @joaomlourenco

**Reply by joaomlourenco** [reply]
@jmcca18 I already know how to make this in a (more or less) easy way.
Just to make it easier, which language are you using to write your dissertation?
And please notice that the FCT-NOVA dissertations do not include the role "Monitor" in the jury… which mean sit is possible that the document is not accepted by the academic services.

**Comment by joaomlourenco** [comment]
@jmcca18 
Download the latest version: 6.9.14!
Add the following to `Config/5_packages.tex` (adapt as necessary):
```latex
\ntaddperson{adviser}(m,m){The monitor name, Monitor, NOVA University Lisbon}
\adviserstring(m,1,m,en):={Monitor}
\adviserstring(m,1,f,en):={Monitor}
\adviserstring(m,2,m,en):={Monitors}
\adviserstring(m,2,f,en):={Monitors}
\adviserstring(m,1,m,pt):={Monitor}
\adviserstring(m,1,f,pt):={Monitora}
\adviserstring(m,2,m,pt):={Monitores}
\adviserstring(m,2,f,pt):={Monitoras}
\adviserorder():={a,c,m}
```
